src/core/issues-notes.js

```diff
@@ -147,59 +174,76 @@ function handleIssues($ins, ghIssues, conf) {
   }
 }
 
-async function fetchIssuesFromGithub({ githubAPI }) {
-  const issues = [];
-  const issueNumbers = [...document.querySelectorAll(".issue[data-number]")]
+async function fetchAndStoreGithubIssues(githubAPI) {
```

Comment:
I still think we should at least write a comment about `AndStore` part before we actually implement it.

Reply:
Will do. We can't land this without caching regardless - as otherwise a few refreshes on some specs causes them to run out of Github request quota.

Reply:
I hope GitHub people allow V4 API public access soon (as it will allow retrieving multiple issue data in a single request)...

Reply:
oh, that's awesome to hear!